src/boot/fathom.js

```diff
@@ -1,16 +0,0 @@
-import { boot } from 'quasar/wrappers';
-
-export default boot(() => {
-    const script = document.createElement('script');
-    script.src = 'https://cdn.usefathom.com/script.js';
-
-    // VMVLEWFD for mainnet
-    // HKAXCRJB for testnet
-    script.dataset.site = 'ISPYEAKT';
-    script.dataset.spa = 'auto';
-    script.defer = true;
-    document.body.appendChild(script);
-
-    console.log('Fathom installed !!!'); // FIXME: remove this line
-});
-
```

Comment:
IDK why this is here. This is what I wrote for OBE but it seems I created here by mistake and forgot to remove it. So I'm doing it now.